<?xml version="1.0"?>
<launch>
    <include file="$(find-pkg-share bitbots_bringup)/launch/base.launch" >
        <arg name="sim" value="true"/>
    </include>

    <param name="use_sim_time" value="false" />

    <node name="dynamic_kick" pkg="bitbots_dynamic_kick" exec="KickNode" output="screen">
        <rosparam command="load" file="$(find-pkg-share bitbots_dynamic_kick)/config/kick_config.yaml" />
        <rosparam command="load" file="$(find-pkg-share bitbots_dynamic_kick)/config/kick_config_unstable.yaml" />
    </node>

    <test pkg="bitbots_dynamic_kick" exec="test_kick_runs.py" test-name="kick_runs" time-limit="300" />
</launch>
